src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfig.java

```diff
@@ -161,6 +164,24 @@ static Filter from(Map<String, Object> properties) {
             final Set<String> ignoredAuditUsers = ImmutableSet.copyOf(getOrDefault(properties, "ignore_users", DEFAULT_IGNORED_USERS));
             final Set<String> ignoreAuditRequests = ImmutableSet.copyOf(getOrDefault(properties, "ignore_requests", Collections.emptyList()));
 
+            boolean invalid = properties.keySet()
```

Comment:
This can be simplified to `ImmutableSet.of("enable_rest", ...).containsAll(properties.keySet())` with `ImmutableSet.of(...)` being `private static Set<String>`